.travis.yml

```diff
@@ -1,9 +1,50 @@
-# Use Docker infrastructure
-sudo: false
-
+---
```

Comment:
@mikewiebe please review this file. Do we need the changes I made last week or will this new config below solve our travis issues?